IdentityCore/src/MSIDExternalAADCacheSeeder.m

```diff
@@ -69,9 +73,17 @@ - (void)seedTokenResponse:(MSIDTokenResponse *)originalTokenResponse
     
     MSID_LOG_WITH_CTX(MSIDLogLevelInfo, requestParameters, @"Beginning external cache seeding.");
     
-    void (^completionBlockWrapper)(void) = ^
+    [[MSIDTelemetry sharedInstance] startEvent:requestParameters.telemetryRequestId eventName:MSID_TELEMETRY_EVENT_EXTERNAL_CACHE_SEEDING];
```

Comment:
does it make it into the aggregated blob?

Reply:
Updated